queries.sql

```diff
@@ -54,4 +54,19 @@ SELECT COUNT(*) AS "Escapee_Animals_Count" FROM animals WHERE escape_attempts =
 SELECT AVG(weight_kg) AS "Average_Animal_Weight" FROM animals;
 SELECT neutered, SUM(escape_attempts) AS "All_Escapes" FROM animals GROUP BY neutered;
 SELECT species, MAX(weight_kg) AS "Maximum Weight", MIN(weight_kg) AS "Minimum Weight" FROM animals GROUP BY species;
-SELECT species, AVG(escape_attempts) AS "Average_Escape" FROM animals WHERE date_of_birth BETWEEN DATE '1990-01-01' AND '2000-12-31' GROUP BY species;
\ No newline at end of file
+SELECT species, AVG(escape_attempts) AS "Average_Escape" FROM animals WHERE date_of_birth BETWEEN DATE '1990-01-01' AND '2000-12-31' GROUP BY species;
+
+/* wednesday */
+
+alter table animals drop column species; /*DELETE COLUMN species*/
+alter table animals add column species_id INT NULL REFERENCES species(id) ON DELETE CASCADE; /*ADD COLUMN species_id, which is a foreign key referencing species table*/
+alter table animals add column owner_id INT NULL REFERENCES owners(id) ON DELETE CASCADE; /*ADD COLUMN owner_id, which is a foreign key referencing owners table*/
+
```

Comment:
- [ ] Everything that creates or changes the tables structure should be saved in the `schema.sql` file. Kindly move this in the respective file.